ntu_rgb_d/msaf_ntu.py

```diff
@@ -182,11 +175,6 @@ def forward(self, tensor_tuple):
         else:
             fm2 = self.visual.features[:13](frames)
 
-        #################################### FIRST msaf
-        # fm2, out5_max ==> fm2, out5_p0 (out5_p1)
-        # fm2, out5_p0 = self.msaf0([fm2, out5_p0])
-        ####################################
-
         # visual
         if self.rgb_net_name == 'resnet':
             # 3rd residual block
```

Comment:
Line 180 to 183 can be moved to after line 174 and line 185 can be combined in line 176 `fm3 = self.visual.features[:15](frames)` This way we can remove the extra if statement